lib/Auth/utils.php

```diff
@@ -69,6 +69,18 @@ public function curl($params, $headers, $serverconfig, $api_endpoint, $http_meth
 		return $body;
 	}
 
+
+	/**
+	 * With this function you can get the authorization token with a service account.
+	 *
+	 * @param $serverconfig
```

Comment:
Please specify the type of the $serviceconfig. (Being an object, key / value whatever)
Is there a defined way to do this?

Reply:
fixed in dae2c98